### PR TITLE
Obsidian2MKdocs fix 404 embedded images

### PR DESCRIPTION
**Pull request by andrezgz**
## Edited
In the Showcase heading of Obsidian to Mkdocs page (`Obsidian2Mkdocs.md`) there are 4 embedded images missing. 

image

They used to be screenshots from [mkdocs_obsidian_publish](https://github.com/Mara-Li/mkdocs_obsidian_publish) repository, but today they are not available anymore.

Since there is an already embedded image from [mkdocs_embed_file_plugins](https://github.com/Mara-Li/mkdocs_embed_file_plugins), `note3.png`, I propose  to add its `demo.gif` to replace the removed embeds.

### CODE REVIEW

**Comment by claremacrae** [comment]
Thanks @andrezgz.

@Mara-Li Do you have any input on this? Is demo.gif likely to remain? Thanks in advance! :-)

File: 06 - Inbox/Obsidian2Mkdocs.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
----
+---https://github.com/Mara-Li/mkdocs_obsidian_template/raw/main/screenshot/image_1.png
```
Comment (claremacrae) [inline 06 - Inbox/Obsidian2Mkdocs.md:1]: I think this bit might be an accidental edit?

**Review by claremacrae — COMMENTED** [review]
Please see the comment about the first line. Thank you!

File: 06 - Inbox/Obsidian2Mkdocs.md (hunk 1 above)
Comment (claremacrae) [reply 06 - Inbox/Obsidian2Mkdocs.md:1]: Oh sorry, I didn't notice the PR was closed...

**Comment by claremacrae** [comment]
@andrezgz Really sorry if any of my comments prompted you to close this. I didn't mean that at all.

It's a really helpful PR that I would love to see merged in to the Hub. Many thanks for making it.

**Comment by andrezgz** [comment]
Hi @claremacrae, it was a rookie mistake... I didn't mean to close it.

**Comment by claremacrae** [comment]
@andrezgz Oh thank you, I hadn’t considered that possibility - even though I did that exact same thing accidentally, early on in my GitHub lifetime :-)

**Comment by andrezgz** [comment]
@claremacrae, I've removed the unintended change on the first line.

My proposal of embedding  `demo.gif` was just because there was a previous image embedded from that same repo... and I've thought it was nice to show it here. If you think it's not OK, just remove it.

Hope this is all that's needed to approve the PR.

Thanks for all the work you do to keep the hub updated.

**Comment by claremacrae** [comment]
> @claremacrae, I've removed the unintended change on the first line.

Thanks!

> My proposal of embedding `demo.gif` was just because there was a previous image embedded from that same repo... and I've thought it was nice to show it here. If you think it's not OK, just remove it.

It's great.

> Hope this is all that's needed to approve the PR.

Very much so. Just about to do it.

Incidentally, by mentioning @Mara-Li above, I was hoping that if demo.gif is ever removed or renamed in future, it will be more likely that this Hub note will be kept uptodate - just a minor thing.

**Review by claremacrae — APPROVED** [review]

**Comment by claremacrae** [comment]
Thanks again @andrezgz - much appreciated! 🎉

**Pull request merged**